modules/deploy/pages/deployment-option/cloud/vpc-byo-gcp.adoc

```diff
@@ -0,0 +1,488 @@
+    "resourcemanager.projects.get",
+    "stackdriver.projects.get",
+    "stackdriver.resourceMetadata.list",
+    "dns.changes.create",
+    "dns.changes.get",
+    "dns.changes.list",
+    "dns.managedZones.list",
+    "dns.resourceRecordSets.create",
+    "dns.resourceRecordSets.delete",
+    "dns.resourceRecordSets.get",
+    "dns.resourceRecordSets.list",
+    "dns.resourceRecordSets.update",
+    "secretmanager.versions.access",
+    "stackdriver.resourceMetadata.write",
+    "storage.objects.get",
+    "storage.objects.list"
+  ],
+}
+EOT
+
+gcloud iam roles create redpanda_gke_utility_role --project=<service-project-id> --file redpanda-gke.role
+
+gcloud projects add-iam-policy-binding <service-project-id> \
+  --member="serviceAccount:redpanda-gke@<service-project-id>.iam.gserviceaccount.com" \
+  --role='projects/<service-project-id>/roles/redpanda_gke_utility_role'
+```
+====
+
+* Redpanda Console service account
++
+.Show commands
+[%collapsible]
+====
+```bash
+gcloud iam service-accounts create redpanda-console \
+  --display-name="Redpanda Console Service Account"
+
+cat << EOT > redpanda-console.role
+{
+  "name": "redpanda_console_secret_manager_role",
+  "title": "Redpanda Console Secret Manager Writer",
+  "description": "Redpanda Console Secret Manager Writer",
+  "includedPermissions": [
+    "secretmanager.secrets.create",
+    "secretmanager.secrets.delete",
+    "secretmanager.secrets.list",
+    "secretmanager.secrets.update",
+    "secretmanager.versions.add",
+    "secretmanager.versions.destroy",
+    "secretmanager.versions.disable",
+    "secretmanager.versions.enable",
+    "secretmanager.versions.list",
+    "iam.serviceAccounts.getAccessToken"
+  ],
+}
+EOT
+
+gcloud iam roles create redpanda_console_secret_manager_role --project=<service-project-id> --file redpanda-console.role
+
+gcloud projects add-iam-policy-binding <service-project-id> \
+  --member="serviceAccount:redpanda-console@<service-project-id>.iam.gserviceaccount.com" \
+  --role='projects/<service-project-id>/roles/redpanda_console_secret_manager_role'
+```
+====
+
+* Redpanda Connectors service account
++
+.Show commands
+[%collapsible]
+====
+```bash
+gcloud iam service-accounts create redpanda-connectors \
+  --display-name="Redpanda Connectors Service Account"
+
+cat << EOT > redpanda-connectors.role
+{
+  "name": "redpanda_connectors_role",
+  "title": "Redpanda Connectors Custom Role",
+  "description": "Redpanda Connectors Custom Role",
+  "includedPermissions": [
+    "resourcemanager.projects.get",
+    "secretmanager.versions.access"
+  ],
+}
+EOT
+
+gcloud iam roles create redpanda_connectors_role --project=<service-project-id> --file redpanda-connectors.role
+
+gcloud projects add-iam-policy-binding <service-project-id> \
+  --member="serviceAccount:redpanda-connectors@<service-project-id>.iam.gserviceaccount.com" \
+  --role='projects/<service-project-id>/roles/redpanda_connectors_role'
+```
+====
+
+* GKE service account
```

Comment:
nit: Can we move this code block up into the above block where we are creating the GKE service account? I think since we collapsed the SA related blocks this now stands out, it looks like a duplicate or something.

Reply:
done!